playbooks/ansible/cluster-glusterfs.yml

```diff
@@ -60,21 +60,21 @@ replicate_cluster_options:
   performance.readdir-ahead: 'on'
   performance.parallel-readdir: 'on'
 
-disperse_cluster_volume: "vol-disperse"
-disperse_cluster_disperse_count: 3
-disperse_cluster_bricks: '/bricks/brick-disp-0/vol,/bricks/brick-disp-1/vol,/bricks/brick-disp-2/vol'
-disperse_cluster_options:
-  features.cache-invalidation: 'on'
-  features.cache-invalidation-timeout: '600'
-  performance.cache-samba-metadata: 'on'
-  performance.stat-prefetch: 'on'
-  performance.cache-invalidation: 'on'
-  performance.md-cache-timeout: '600'
-  network.inode-lru-limit: '200000'
-  performance.nl-cache: 'on'
-  performance.nl-cache-timeout: '600'
-  performance.readdir-ahead: 'on'
-  performance.parallel-readdir: 'on'
+# disperse_cluster_volume: "vol-disperse"
+# disperse_cluster_disperse_count: 3
+# disperse_cluster_bricks: '/bricks/brick-disp-0/vol,/bricks/brick-disp-1/vol,/bricks/brick-disp-2/vol'
+# disperse_cluster_options:
+#   features.cache-invalidation: 'on'
+#   features.cache-invalidation-timeout: '600'
+#   performance.cache-samba-metadata: 'on'
+#   performance.stat-prefetch: 'on'
+#   performance.cache-invalidation: 'on'
+#   performance.md-cache-timeout: '600'
+#   network.inode-lru-limit: '200000'
+#   performance.nl-cache: 'on'
+#   performance.nl-cache-timeout: '600'
+#   performance.readdir-ahead: 'on'
+#   performance.parallel-readdir: 'on'
```

Comment:
Shouldn't we just delete this ?

If we want to re-add it in the future, it can be recovered from git history.

Reply:
Fine.